Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -17,9 +17,16 @@ public class CBPurchase: NSObject {
     private var authenticationManager = CBAuthenticationManager()
     var productRequest: SKProductsRequestFactory = SKProductsRequestFactory()
 
-    private var restoredPurchasesCount = 0
+    var restoredPurchasesCount = 0
     private var activeProduct: SKProduct?
     var customer: CBCustomer?
+    
+     var restoreResponseHandler: ((Result<[InAppSubscription], RestoreError>) -> Void)?
+     var result: ReceiptResult<String>?
+     var refreshHandler: RestoreResultCompletion<String>?
+
+     public var includeInActiveProducts = false
+     var restoreHandlerisActive = false
 
     // MARK: - Init
     private override init() {
```

Comment:
`func stopPaymentQueueObserver() {
        SKPaymentQueue.default().remove(self)
    }`
we have this method  already` in SDK but it didn't used any where we have to de-init when app is terminated

Reply:
i have addressed all comments and updated code accordingly , can you confirm on this ? shall i add de-init in this PR only couple i lines it would need for this i can add please confirm ?

Reply:
Lets add deinit as part of this please.